drivers/gpu/drm/panel/Kconfig

```diff
@@ -812,6 +812,14 @@ config DRM_PANEL_STARTEK_KD070FHFID015
 	  with a resolution of 1024 x 600 pixels. It provides a MIPI DSI interface to
 	  the host, a built-in LED backlight and touch controller.
 
+config DRM_PANEL_TD4320_BOEPLUS
+	tristate "TD4320 BOEPLUS DSI panel"
+	depends on OF
+	depends on DRM_MIPI_DSI
+	depends on BACKLIGHT_CLASS_DEVICE
+	help
+	  Say Y here if you want to enable support for TD4320.
```

Comment:
```
Say Y or M here if you want to enable support for TD4320
panel used in xiaomi-lavender phone.
```
maybe?

Reply:
Other panels are not mentioning M

Reply:
Ok, don't mention M, still description can be a lot better, new suggestion:
```
Say Y here if you want to enable support for BOE TD4320
panel used in Xiaomi Redmi Note 7 phone.
```